lua/user/autocommands.lua

```diff
@@ -3,6 +3,12 @@ local M = {}
 local create_aucmd = vim.api.nvim_create_autocmd
 
 M.config = function()
+  vim.api.nvim_create_autocmd("ColorScheme", {
```

Comment:
if we can use this, then remove this please:
https://github.com/abzcoding/lvim/blob/420911a77452ceb3d4c2a27726d83192f2cb3f3c/after/plugin/highlights.lua#L1

Reply:
Fixed in eafcaeb